python/lsst/sims/utils/stellarMags.py

```diff
@@ -0,0 +1,102 @@
+import numpy as np
+
+__all__ = ['stellarMags']
+
+
+def calcWDColors():
+    """
+    Calculate a few example WD colors. Values to go in stellarMags(). Here in case
+    values need to be regenerated (different stars, bandpasses change, etc.)
+    """
+
+    from lsst.utils import getPackageDir
+    import os
+    from lsst.sims.photUtils import Bandpass, Sed
+
+    names = ['WD_25000_80', 'HeWD_11000_80', 'WD_3050_80']
+    fns = ['bergeron_25000_80.dat_25000.gz', 'bergeron_He_11000_80.dat_11000.gz', 
+             'bergeron_3000_80.dat_3050.gz']
+    wdDir = os.path.join(getPackageDir('sims_sed_library'), 'starSED/wDs/')
```

Comment:
Since sims_utils depends on neither sims_sed_library nor on throughputs, could you put these calls to getPackageDir inside of try/except blocks that add a reassuring message for the user.  getPackageDir's failure message is pretty explanatory, but it might be nice to add a sentence explaining to the user that it isn't alarming that throughputs or sims_sed_library weren't there; they just need to be explicitly set up.